src/AddressRegistry.Api.BackOffice/Validators/AddressProposeRequestValidator.cs

```diff
@@ -32,7 +32,7 @@ public AddressProposeRequestValidator(SyndicationContext syndicationContext)
                 .WithErrorCode(ValidationErrors.Address.BoxNumberInvalid);
 
             RuleFor(x => x.PositieGeometrieMethode)
```

Comment:
Fyi @ArneD , I left this validation to satisfy following test https://github.com/Informatievlaanderen/address-registry/blob/main/test/AddressRegistry.Tests/BackOffice/Api/WhenProposingAddress/GivenInvalidRequest.cs#L151